DEVELOPING_ON_MODULAR.md

```diff
@@ -0,0 +1,93 @@
+# Developing on Modular Volttron
+
+
+The following documentation steps through the process of setting up your environment to start
+developing on modular VOLTTRON.
+
+There are requirements in order to follow this README. The following requirements should be installed system-wide, 
+so they are available to be used throughout the README.
+
+* Poetry 1.2.2
+
+## Installing poetry
+See [poetry installation](https://github.com/eclipse-volttron/volttron-core/tree/develop#poetry)
+
+
+# Development
+
+##  New Agent 
+* Please use the tool [copier-poetry-volttron-agent](https://github.com/VOLTTRON/copier-poetry-volttron-agent) to create a new agent. 
+
+## Existing Agent
+* Fork the Agent repo on Github
+* Clone the repo in your local environment
+* Setup the environment using poetry
+
+```shell
+poetry install 
+```
+
+* Create a working branch off of 'develop'
```

Comment:
Why are these three bulleted.

Reply:
They are bulleted to make it clear that to the user on how to start developing on an existing Agent; it tries to mirror the TDD process of working on legacy code.

Reply:
Ahh perhaps a header to say "the recommended development path is.... or something.  The bullets seem to spring up almost out of nowhere to me.

Reply:
Changed.